UniWatch/Controllers/StudentController.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Web;
+using System.Web.Mvc;
+using UniWatch.DataAccess;
+using UniWatch.Models;
+using UniWatch.ViewModels;
+
+namespace UniWatch.Controllers
+{
+    public class StudentController : Controller
+    {
+        private IDataAccess _dataAccess;
+
+        public StudentController() : this(new DataAccess.DataAccess())
+        { }
+
+        public StudentController(IDataAccess dataAccess)
+        {
+            _dataAccess = dataAccess;
+        }
+
```

Comment:
To help with readability, group the methods by their name, rather than having them strewn across the controller.

**i.e.** Index (GET), Enroll (GET, POST), UnEnroll (GET, POST), Search